src/Scrollable/Scrollable.js

```diff
@@ -138,8 +138,8 @@ var Scrollable = {
 				// default, we short-circuit the action because the child is
 				// potentially not in view, and focusing would cause it to
 				// scroll it into view for no reason apparent to the user.
-				// Instead, we should focus a child we know to be visible.
-				if (this.eventIsFromScrollingChild(event)) {
+				// Instead, we should focus a child we know to be visible.				
+				if (this.eventIsFromScrollingChild(event) && !this.isVisibleChild(event.originator) ) {
```

Comment:
For consistency & safety, it may be best to pass `event` as the argument of `isVisibleChild()` here. In doing that, the method name `eventIsFromVisibleChild()` could make more sense.